recipe/meta.yaml

```diff
@@ -13,33 +7,35 @@ package:
   version: {{ cuda_compiler_version }}
 
 build:
-  number: 25
+  number: 26
   skip: {{ not (compiler("cuda").startswith("nvcc") and cuda_compiler_version == "11.8") }}
 
 outputs:
-  - name: nvcc_{{ cross_target_platform }}
-    script: linux/install_nvcc.sh     # [linux]
-    script: windows/install_nvcc.bat  # [win]
+  - name: {{ cuda_compiler }}_{{ cross_target_platform }}
```

Comment:
This fixes a re-render issue where re-rendering was not adding `cuda-compiler` and in some cases disabling CI

Note the package name is exactly the same as before